news/369.bugix

```diff
@@ -0,0 +1,2 @@
+- Fixes the "by" translation and the date issues for document byline viewlet
```

Comment:
```suggestion
Fixes the "by" translation and the date issues for document byline viewlet
```

The dash in front is automatically added when I generate the changelog, you should not add it here, otherwise it will be double.